db/migrate/20190701174401_add_http_proxy_policy_to_repository_root.rb

```diff
@@ -0,0 +1,6 @@
+class AddHttpProxyPolicyToRepositoryRoot < ActiveRecord::Migration[5.2]
+  def change
+    add_column :katello_root_repositories, :http_proxy_policy, :string,
+      default: Katello::RootRepository::NO_DEFAULT_HTTP_PROXY
```

Comment:
this looks good, we probably want to import the 'skip global proxy' column that is currently there and drop it

Reply:
I was wondering if we need to add a migration that sets the proxy policy based on what the "ignore global proxy" column value is? I wasn't sure how commonly this setting is used or how important it is to carry that information forward.

Reply:
yep, i think we do.  Its common enough to carry its intent forward.

Reply:
Should the default be, instead of 'No Default",set "use global default", and then set the records that already have the "ignore" column as true set to to "no default"?

Reply:
yep, exactly!  The current default is 'use global default' (which can mean no proxy), so all repos on the new model should be set to that unless ignore_global_proxy is set to true

Reply:
That's what I thought. As it is said, so let it be written.